js/main.js

```diff
@@ -244,12 +255,19 @@ menuIcon.addEventListener('click', () => {
   }
 });
 
-const form = document.getElementById('form');
-const email = document.getElementById('email');
-const messageBox = document.getElementById('error-message');
+// Set Data To local Storage
+function saveTolocalStorage() {
+  const myFormSubmit = {
+    name: nameElement.value,
+    email: emailElement.value,
+    message: messageElement.value,
+  };
+  localStorage.setItem('myFormSubmit', JSON.stringify(myFormSubmit));
```

Comment:
- [ ] The `saveTolocalStorage` function is working fine, but not how it is supposed to. The form is saved after the user submits it when it should be saved after the user changes the content of any input field, so if the user accidentally reloads the page, the content is still there. If the user submits the form, there is no purpose for reloading the same information to the form since it was already submitted. Kindly change where this function is implemented in order to do that as this is the project requirement.
 -
<img width="882" alt="image" src="https://user-images.githubusercontent.com/61919617/195632608-a4a25055-927d-47b6-91d9-22b518dfa568.png">